pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyGovernanceDao.java

```diff
@@ -64,6 +68,20 @@ default GovernancePolicy createPolicy(GovernancePolicy policy) {
     @UseRowReducer(PolicyWithAgeOfMajorityRuleReducer.class)
     Optional<GovernancePolicy> findPolicyByStudyGuid(@Bind("studyGuid") String studyGuid);
 
+    /**
+     * Deletes the policy and any related rules.
+     */
+    default void deletePolicy(long policyId) {
+        var studyGovernanceSql = getStudyGovernanceSql();
+        int numRulesDeleted = studyGovernanceSql.deleteRulesForPolicy(policyId);
```

Comment:
There's cascade delete in `age_of_majority_rule` so deleting the policy should delete the rules. I recommend not doing this extra work, unless we wanted to log how many rules were deleted.

Reply:
Sweet!